packages/admin/cms-admin/src/index.ts

```diff
@@ -114,6 +114,11 @@ export { useSiteConfig } from "./sitesConfig/useSiteConfig";
 export { useSitesConfig } from "./sitesConfig/useSitesConfig";
 export { AzureAiTranslatorProvider } from "./translation/AzureAiTranslatorProvider";
 export { CurrentUserInterface, CurrentUserProvider, useCurrentUser, useUserPermissionCheck } from "./userPermissions/hooks/currentUser";
+export { UserBasicData } from "./userPermissions/user/basicData/UserBasicData";
+export { ImpersonationButton, StopImpersonationButton } from "./userPermissions/user/ImpersonationButtons";
+export { UserPermissionsPanel } from "./userPermissions/user/permissions/PermissionsPanel";
+export { UserPageToolbar } from "./userPermissions/user/UserPageToolbar";
+export { UserGrid } from "./userPermissions/UserGrid";
```

Comment:
in the exports you unfortunately loose the module (folder) information, the exports should be prefixed with UserPermissions or so